test/unit/util_spec.js

```diff
@@ -185,6 +185,16 @@ describe("util", function () {
       const str = "string\x00With\x00Null\x00Chars";
       expect(removeNullCharacters(str)).toEqual("stringWithNullChars");
     });
+
+    it("should modify string with non-displayable characters", function () {
+      const str = Array.from(Array(32).keys())
+        .map(x => String.fromCharCode(x))
+        .join("a");
```

Comment:
Is this string perhaps missing the last "a" character, in order for it to be consistently formatted? Currently this code gives:
```
\u0000a\u0001a\u0002a\u0003a\u0004a\u0005a\u0006a\u0007a\u0008a	a
a\u000ba\u000ca
a\u000ea\u000fa\u0010a\u0011a\u0012a\u0013a\u0014a\u0015a\u0016a\u0017a\u0018a\u0019a\u001aa\u001ba\u001ca\u001da\u001ea\u001f
```
whereas I believe that you probably want to use e.g. 
```javascript
const str = [...Array(32).keys()]
  .map(x => String.fromCharCode(x) + "a")
  .join("");
```
since that'd give you:
```
\u0000a\u0001a\u0002a\u0003a\u0004a\u0005a\u0006a\u0007a\u0008a	a
a\u000ba\u000ca
a\u000ea\u000fa\u0010a\u0011a\u0012a\u0013a\u0014a\u0015a\u0016a\u0017a\u0018a\u0019a\u001aa\u001ba\u001ca\u001da\u001ea\u001fa
```